src/screencast_keys/preferences.py

```diff
@@ -402,8 +408,9 @@ def draw(self, context):
 
             layout.label(text="Experimental:")
             col = layout.column()
-            col.prop(self, "get_event_aggressively")
-            col.enabled = False if c_structures.NOT_SUPPORTED else True
+            col.prop(self, "load_on_startup")
+            if c_structures.NOT_SUPPORTED == False:
+                col.prop(self, "get_event_aggressively")
```

Comment:
Please remain the **Get Event Aggressively** visible if the option is not supported.

Reply:
Turned this back and moved `load_on_startup` to UI section.